### PR TITLE
Add https and wireguard on ISO

### PR DESCRIPTION
**Pull request by pablomendezroyo**

### CODE REVIEW

File: scripts/dappnode_install.sh [hunk 1]
```diff
@@ -38,6 +38,14 @@ WGET="wget -q --show-progress --progress=bar:force"
 SWGET="wget -q -O-"
 IPFS_ENDPOINT=${IPFS_ENDPOINT:-"http://ipfs.io"}
 
+detect_installation_type() {
+    # Check for old and new location of iso_install.log
+    if [ -f "${DAPPNODE_DIR}/iso_install.log" ] || [ -f "${DAPPNODE_DIR}/logs/iso_install.log" ]; then
```
Comment (dapplion) [inline scripts/dappnode_install.sh:43]: The existance of this file is what triggers and ISO installation? Could we use a flag file specifically for this purpose?

Comment (pablomendezroyo) [reply scripts/dappnode_install.sh:43]: yeah that makes much more sense

Comment (pablomendezroyo) [reply scripts/dappnode_install.sh:44]: I would suggest to implement this in another issue https://github.com/dappnode/DAppNode_Installer/issues/144

File: .github/workflows/linter.yml [hunk 2]
```diff
@@ -5,10 +5,7 @@ name: Lint Code Base
 # https://help.github.com/en/articles/workflow-syntax-for-github-actions
 #
 on:
-  push:
-    branches-ignore: [master]
-  pull_request:
-    branches: [master]
+  workflow_dispatch:
```
Comment (dapplion) [inline .github/workflows/linter.yml:8]: Why changing the triggers for this action?

Comment (pablomendezroyo) [reply .github/workflows/linter.yml:8]: It is annoying to have linter on every push/pr since it displays too many recomendations for the code

Comment (dapplion) [reply .github/workflows/linter.yml:8]: Linters should run always. This annoyance should be motivation to fix the warnings

**Review by dapplion — APPROVED** [review]

**Review by dapplion — APPROVED** [review]

**Pull request merged**